proposals/2024-06-14_mixxx_e_v.md

```diff
@@ -0,0 +1,566 @@
+   Vorsitzenden bei dessen Abwesenheit und überprüft die Beschlüsse des
+   Vorstands und der Mitgliederversammlungen. Der Schatzmeister verwaltet die
+   Finanzen des Vereins, führt die Kassenbücher und die Mitgliederliste.
+3. Der Vorstand beschließt über alle Vereinsangelegenheiten, soweit sie nicht
+   eines Beschlusses der Mitgliederversammlung bedürfen. Er führt die
+   Beschlüsse der Mitgliederversammlung aus.
+4. Jedes Vorstandsmitglied hat das Recht, den Verein zu vertreten.
+5. Im Falle des Ausscheidens eines Vorstandsmitglieds während der Amtszeit
+   bestimmt der Vorstand ein kommissarisches Vorstandsmitglied. Das
+   kommissarische Vorstandsmitglied bleibt bis zur nächsten
+   Mitgliederversammlung im Amt. Über eine endgültige Nachfolge im
+   Vorstand entscheidet die Mitgliederversammlung.
+6. Im Falle des Ausschlusses eines Vorstandsmitglieds gemäß §4 (5) wird dieses
+   durch ein kommissarisches Vorstandsmitglied ersetzt.
+7. Der Vorstand kann zur Führung der laufenden Geschäfte einen Geschäftsführer
+   bestellen.
+8. Der Geschäftsführer nimmt an den Mitgliederversammlungen und
+   Vorstandssitzungen teil.
+9. Der Vorstand kann die Satzung aus formalen Gründen ändern, wenn von den
+   Behörden verlangt wird. Über die Änderung wird in der darauffolgenden
+   Mitgliederversammlung abgestimmt.
+
+## 9. Geschäftsordnung
+
+Die Geschäftsordnung regelt über den Rahmen der Satzung hinausgehende Details
+der Vereinsarbeit. Der Vorstand setzt die Geschäftsordnung nach der Zustimmung
+der Kernmitglieder in einer Internetabstimmung in Kraft.
+
+## 10. Protokolle
+
+Alle Beschlüsse des Vorstandes und der Mitgliederversammlung werden schriftlich
+in deutscher Sprache dokumentiert und stehen den Mitgliedern zur Einsichtnahme
+und zur Prüfung der Beschlüsse durch das Registergericht zur Verfügung.
+
+## 11. Internetabstimmung
+
+Der Verein kann Abstimmungen über das Internet durchführen. Eine
+Internetabstimmung ist gültig nach einer Ankündigung per E-Mail oder im
+obligatorischen Gruppenchat der Kernmitglieder und wenn ein Quorum von
+mindestens 1/2 der Kernmitglieder teilnimmt. Der Abstimmungszeitraum muss
+mindestens zwei Wochen nach der Ankündigung betragen, um sicherzustellen,
+dass die Mitglieder genügend Zeit haben, ihre Stimme abzugeben.
+
+## 12. Tarifverträge
+
+Vor der Anstellung hauptamtlich Beschäftigter regelt der Vorstand deren
+Bezahlung in der Geschäftsordnung.
+
+## 13. Vereinsfinanzierung
+
+1. Der Verein wird durch Mitgliedsbeiträge, öffentliche Zuschüsse, Spenden
+   und sonstige Zuwendungen Dritter finanziert.
+2. Die Höhe der Mitgliedsbeiträge wird durch die Mitgliederversammlung
+   entschieden.
+3. Bei Auflösung des Vereins oder bei Wegfall steuerbegünstigter Zwecke fällt
+   das Vereinsvermögen an den K Desktop Environment e.V. Berlin, der es
+   ausschließlich und unmittelbar für gemeinnützige Zwecke im Sinne dieser
+   Satzung zu verwenden hat.
```

Comment:
KDE is aware of that, right?

Reply:
Not yet. I want to propose it and if we agree, ask them.